int64 marknumber
bool permanent
string path
---
bool result
